src/SarifTests/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Runtime.CompilerServices;
```

Comment:
These files are still present in your enlistment. You must delete the SarifTests directory.